<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>661</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>26</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
border: 4px solid;
border-color: rgb(72, 57, 33);

background-color: qlineargradient(spread:pad, x1:0.482587, y1:1, x2:0.512, y2:0, stop:0 rgba(176, 128, 60, 255), stop:1 rgba(221, 162, 91, 255));

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>6</y>
      <width>346</width>
      <height>583</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid;
border-color: rgb(72, 57, 33);
border-radius: 20px;
background-color: rgb(184, 137, 71);
background-color: qlineargradient(spread:pad, x1:0.592, y1:1, x2:0.532, y2:0, stop:0.039801 rgba(214, 137, 68, 255), stop:0.512438 rgba(201, 134, 74, 251), stop:1 rgba(235, 179, 101, 255));</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>18</y>
       <width>301</width>
       <height>112</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
     <property name="text">
      <string>Название сорта:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="degree_of_roasting">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>129</y>
       <width>316</width>
       <height>112</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
     <property name="text">
      <string>Степень обжарки:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="shredded">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>246</y>
       <width>316</width>
       <height>112</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
     <property name="text">
      <string>Вид:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="taste">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>354</y>
       <width>316</width>
       <height>112</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
     <property name="text">
      <string>Описание вкуса:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="price">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>456</y>
       <width>136</width>
       <height>112</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
     <property name="text">
      <string>Цена:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="package_volume_grams">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>459</y>
       <width>166</width>
       <height>112</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
     <property name="text">
      <string>Объем упаковки:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>606</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton{border-radius: 20}</string>
    </property>
    <property name="text">
     <string>Скрыть</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
